darwin-types/src/game-objects/Consumable.ts

```diff
@@ -0,0 +1,10 @@
+import { GameObject } from './GameObject';
+import { State } from '../State';
+import { UserContext } from '../UserContext';
+
+export type Consume = (state: State, userContext: UserContext) => State;
+
+export interface Consumable extends GameObject {
+  consume: Consume;
```

Comment:
This weakens the strengths of TypeScript. It's available on the backend but not on the frontend - is this what we want?
Maybe it would make sense to refactor our typings now as the monorepo setup is in place.